### PR TITLE
logo-type in header, move sort order dropdown, refs #2391

### LINKED ISSUE #2391: Include the full logotype on mobile
**The problem**
Currently, we do not show the full logotype within the mobile primary navigation. This means we're missing an important branding opportunity when new users come from a google search or social link.

screen shot 2018-01-23 at 11 16 03 am

**The solution**
Display the full steemit logotype for all screen sizes both for authenticated and non-authenticated users.

screen shot 2018-01-23 at 11 19 31 am

_Home page_

To implement this, we'll need to remove the content filter within the primary navigation on the home page. Instead, we'll add a dropdown to the page title. 

screen shot 2018-01-23 at 11 19 20 am

----

Other elements and icons also need to be updated but I'll create another ticket for that.

### PR DESCRIPTION
**Pull request by gl2748**
closes #2391 
New header.

### CODE REVIEW

**Comment by bnchdrff** [comment]
search & post buttons should have the same outline style; currently the search button has an inner shadow

screenshot from 2018-03-02 15-00-54

**Comment by bnchdrff** [comment]
and then if you're logged in w/ the default avi, you get another slightly different circle button:

screenshot from 2018-03-02 15-03-12

**Comment by bnchdrff** [comment]
in Firefox, the header overflows <640px:

screen shot 2018-03-02 at 15 06 03
screen shot 2018-03-02 at 15 05 54

**Comment by bnchdrff** [comment]
i think it's funny looking how the blog/list mode toggle is slightly shorter than the two select dropdowns (tags/trending) -- it kinda makes it look like two squinting eyes with a small crinkled nose in the middle

**Comment by bnchdrff** [comment]
on iphone se, buttons overlap logo:

screenshot from 2018-03-06 11-39-40

File: src/app/components/elements/SearchInput/SearchInput.story.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import SearchInput from './index';
+import { Center } from '../Tooltip.story';
+
+storiesOf('Elements', module)
+    .addDecorator(withKnobs)
```
Comment (bnchdrff) [inline src/app/components/elements/SearchInput/SearchInput.story.jsx]: vestigal knobs

File: src/app/components/elements/SearchInput/index.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+const SearchInput = ({ type }) => {
+    return (
+        <span>
+            <form
+                className={'search-input'}
+                action="/static/search.html"
+                method="GET"
+            >
+                <svg
```
[the remark this answers is not in the document]
Comment (gl2748) [reply src/app/components/elements/SearchInput/index.jsx]: I don't really like what the icons component is doing. It's not very clear, and I have found it hard to work with.

[the remark this answers is not in the document]
Comment (bnchdrff) [reply src/app/components/elements/SearchInput/index.jsx]: cool, yeah it's a mess i agree

File: src/app/utils/getPageTitle.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import tt from 'counterpart';
+import normalizeProfile from 'app/utils/NormalizeProfile';
+
+const getPageTitle = (route, account_meta) => {
+    let page_title = route.page;
+    let sort_order = '';
+    let topic = '';
+    let user_name = null;
+    if (route.page === 'PostsIndex') {
```
Comment (bnchdrff) [inline src/app/utils/getPageTitle.js]: a switch statement would be a little easier to read & maintain

File: src/app/redux/UserReducer.js [hunk 4]
```diff
@@ -394,3 +404,13 @@ export const uploadImage = payload => ({
     type: UPLOAD_IMAGE,
     payload,
 });
+
+export const showSidePanel = () => ({
+    type: SHOW_SIDE_PANEL,
+});
+
+export const hideSidePanel = () => {
```
Comment (bnchdrff) [inline src/app/redux/UserReducer.js]: use shortand syntax like you do in showSidePanel above

**Pull request merged**